### ISSUE TITLE
Flextable and Pandoc

### ISSUE THREAD

**Issue by jlgraves-ubc** [quarto-dev/quarto-cli#4896]
### Bug description

Hi all,

I am trying to adapt some of my old RMD templates, and I have a strange issue.  When I render a .qmd file to DOCX, the `flextable` package complains that `pandoc version >= 2 required for printing flextable in docx`

The thing is, quarto reports that `quarto pandoc --version` is `pandoc 2.19.2`

I've tried reinstalling the package, and fiddling around, but to no avail.  I know the easy solution would be to not use `flextable` but it's embedded in all of my fancy templates.

Interestingly, this works just fine in RStudio with the original RMarkdown version of the file.

It seems to fail with any .qmd file:

```
---
title: test
output:
    docx: default
---

```{r}
library(flextable)
library(readr)
flextable(read_csv("a.csv") )

```
```

Quarto is Version: 1.2.247

### Checklist

- [X] Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [X] Please [format your issue](https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us) so it is easier for us to read the bug report.
- [X] Please document the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [X] Please document the operating system you're running. If on Linux, please provide the specific distribution.
- [X] Please provide the output of `quarto check` so we know which version of quarto and its dependencies you're running.

**Comment by cscheid** [comment]
We support `flextable`, and this should work without a problem.

>  Please provide the output of `quarto check` so we know which version of quarto and its dependencies you're running.

I don't see the output of `quarto check` in your message. Can you actually provide this output? Not the quarto version, but the actual full output of the command.

**Comment by jlgraves-ubc** [comment]
```

[>] Checking Quarto installation......OK
      Version: 1.2.247
      Path: C:\Program Files\Quarto\bin
      CodePage: 1252

[>] Checking basic markdown render....OK

[>] Checking Python 3 installation....OK
      Version: 3.11.0
      Path: C:/Python/python.exe
      Jupyter: 5.0.0
      Kernels: ir, python3

(|) Checking Jupyter engine render....0.00s - Debugger warning: It seems that frozen modules are being used, which may
(/) Checking Jupyter engine render....0.00s - make the debugger miss breakpoints. Please pass -Xfrozen_modules=off
0.00s - to python to disable frozen modules.
0.00s - Note: Debugging will proceed. Set PYDEVD_DISABLE_FILE_VALIDATION=1 to disable this validation.
0.00s - Debugger warning: It seems that frozen modules are being used, which may
0.00s - make the debugger miss breakpoints. Please pass -Xfrozen_modules=off
0.00s - to python to disable frozen modules.
0.00s - Note: Debugging will proceed. Set PYDEVD_DISABLE_FILE_VALIDATION=1 to disable this validation.
[>] Checking Jupyter engine render....OK

[>] Checking R installation...........OK
      Version: 4.2.1
      Path: C:/PROGRA~1/R/R-42~1.1
      LibPaths:
        - C:/Users/Jonathan Graves/AppData/Local/R/win-library/4.2
        - C:/Program Files/R/R-4.2.1/library
      rmarkdown: 2.17

[>] Checking Knitr engine render......OK

```

**Comment by cscheid** [comment]
It looks like your packages are out of date; can you try upgrading them? I wonder if this is ultimately causing the trouble (I can't reproduce your issue on my side; we test with flextable constantly). You're also running an older quarto version. Any change you can upgrade that?

**Comment by jlgraves-ubc** [comment]
No luck!  I updated all my R packages, including flextable, and my Quarto install.  I'm going to keep trying a few different things, but it might be some super idiosyncratic issue with my installation.

**Comment by cscheid** [comment]
I'm sorry to hear that.

> I updated all my R packages, including flextable, and my Quarto install

Is it possible that you have two separate R installations, so that you're updating one, and quarto is seeing the other?

For example, what happens if you render the following .qmd?

````
---
format: html
title: version check
---

```{r}
library(flextable)
sessionInfo()
```
````

in there, you'll find (in my case)

```
other attached packages:
[1] flextable_0.8.3
```

Then run the same commands in the same R setup you used to upgrade your packages. My theory is that there'll be a difference.

**Comment by jlgraves-ubc** [comment]
The only difference is that one has the `yaml` package included when run from Quarto render; otherwise, they look identical.  I'll look into this a little more.  I have R install normally, and then via `conda` but Quarto isn't point to that one which is a different version.  I'll poke around a little more to see if there's a conflict in my configuration.

**Comment by cscheid** [comment]
You might find more information with these commands:

```r
Sys.getenv() # env variables
.libPaths() # library paths 
```

**Comment by jlgraves-ubc** [comment]
Yeah, very weird; all of the items I can identify which are common between the two outputs are identical.   I'm going to try purging my R installs this weekend, once I'm done with work for the week, then reconfiguring everything.

**Comment by arthurgailes** [comment]
Same problem here with a barebones reproduction. On one setup, it works fine; on another, it breaks. Seems to be caused by the update from 1.2.269 to 1.2.313.

Working build:

- Quarto 1.2.269
- pandoc 2.19.2
- flextable_0.9.0

Broken build:

- Quarto 1.2.313
- pandoc 2.19.2
- flextable_0.9.0

```
---
format: docx
---


# omitting the duplicate code wrapper for GH formatting
library(flextable)

ft <- flextable(head(iris))
ft
```

**Comment by cderv** [comment]
@arthurgailes @jlgraves-ubc can you try with latest RC 1.3 (https://quarto.org/docs/download/) 

@arthurgailes I can't reproduce your example. This works fine. 1.3 has some improvments on tables so possibly this has been fixed.

**Comment by arthurgailes** [comment]
@cderv I believe my issue here is actually #5104.
